site/styles/_pages.scss

```diff
@@ -69,9 +71,10 @@
     min-height: 740px;
   }
 }
+.account-page {
```

Comment:
I'm curious why this needs to exist... if the signin/signup pages can exist with just the `form-page` class, why can't the account page?

Reply:
Yup. You're right. It doesn't need to exist. I think I previously had `form-page` as `margin: 0 auto` set and it was causing the account page to bug out. Removed in the latest commit.